.gitignore

```diff
@@ -0,0 +1,36 @@
+*/target/
+!.mvn/wrapper/maven-wrapper.jar
+
+# Have stuff you don't want to commit but still find useful to hang around locally?
```

Comment:
I would expect this file to match https://github.com/department-of-veterans-affairs/health-apis-clinical-fhir/blob/master/.gitignore